Pipfile

```diff
@@ -48,5 +48,7 @@ RTFDE = "==0.0.2"
 win_unicode_console = "==0.5"
 wxPython = "==4.1.1"
 pytest = "==7.0.1"
+warcio = "*"
+requests = "*"
 
```

Comment:
These also need to be added to `setup.py`. We should also discuss possibly pinning a version.

Reply:
and `requirements.txt`

Reply:
Pin to > latest major version as discussed.